message.go

```diff
@@ -23,8 +23,14 @@ const (
 	MediaDocument MediaType = "WhatsApp Document Keys"
 )
 
-func (wac *Conn) Send(msg interface{}) (string, error) {
+// Since there is no optional variable in go, using this as a workaround
+func (wac *Conn) Send(msg interface{}, medias ...Media) (string, error) {
```

Comment:
Why not another function which sends message with a specific media?
So there you only need verifying if media is valid and really separating uploading media from sending messages.

Reply:
Just to keep it one place because the original author is using conditions to differentiate between media instead of creating mutliple functions